include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -536,7 +524,7 @@ struct __result_and_scratch_storage_impl : __result_and_scratch_storage_base
     using __accessor_t =
         sycl::accessor<_T, 1, _AccessMode, __dpl_sycl::__target_device, sycl::access::placeholder::false_t>;
 
-    _ExecutionPolicy __exec;
+    mutable sycl::queue __q;
```

Comment:
I declared this `sycl::queue __q` as `mutable` because their non-const method `sycl::queue::memcpy` using in const-methods of this `struct __result_and_scratch_storage`